### PR TITLE
H/E Pipes glow

### PR DESCRIPTION
**Pull request by Mazianni**
Radiator pipes now emit light and have a faint proper glow to them depending on temperature.
https://cdn.discordapp.com/attachments/678826323728400394/848542092765823056/unknown.png

### CODE REVIEW

File: code/modules/atmospherics/he_pipes.dm [hunk 1]
```diff
@@ -100,6 +103,10 @@
 				for (var/thing in animate_targets)
 					var/atom/movable/AM = thing
 					animate(AM, color = rgb(h_r, h_g, h_b), time = 20, easing = SINE_EASING)
+				filters[1].color = rgb(h_r, h_g, h_b)
```
Comment (MistakeNot4892) [inline code/modules/atmospherics/he_pipes.dm]: Not sure I like this. Can you store the `filter()` return somewhere instead?

Comment (Lohikar) [reply code/modules/atmospherics/he_pipes.dm]: This also isn't updating the color of mimics.

Comment (Mazianni) [reply code/modules/atmospherics/he_pipes.dm]: The mimics don't have any filters - the filters are on the base H/E pipe itself, so it doesn't update the mimic filters as intended.

**Review by MistakeNot4892 — APPROVED** [review]

File: code/_helpers/visual_filters.dm [hunk 2]
```diff
@@ -35,4 +35,23 @@
 	if(thing)
 		LAZYREMOVE(filter_data, filter_name)
 		filters -= thing
-		update_filters()
\ No newline at end of file
+		update_filters()
+
+/// Animate a given filter on this atom. All params after the first are passed to animate().
```
Comment (Lohikar) [inline code/_helpers/visual_filters.dm:40]: This comment didn't get updated to match the new proc version.

**Pull request merged**